src/schema/datatypes/anat.yaml

```diff
@@ -0,0 +1,45 @@
+groups:
```

Comment:
To continue from https://github.com/bids-standard/bids-specification/pull/475#discussion_r428050851

Does this work?

Reply:
I don't think this top level `groups:` is needed unless you are planing to come up with another key in addition to `groups` at this level.

Reply:
Would it be better to have it as a list of dictionaries _or_ as separate "documents" in the same file (see [here](https://gettaurus.org/docs/YAMLTutorial/#YAML-Multi-Documents))?